### DISCUSSION TITLE
Trouble dependency for telescope

### DISCUSSION THREAD

**Discussion by diocletiann** [LazyVim/LazyVim#3526] [Q&A] [answered]
It seems a recent update of LazyVim made trouble required for telescope, I had it disabled but now I get:
`...l/share/nvim/lazy/LazyVim/lua/lazyvim/plugins/editor.lua:241: module 'trouble.sources.telescope' not found:`

Is it possible to remove this dependency? Thanks.

**Comment by dpetka2001** [comment, marked as answer]
It's solved by #3518 but it will be available on next release. If you follow HEAD then you shouldn't have any problems.

**Reply by diocletiann** [reply]
Thanks, HEAD works!
